apollo-api/gradle.properties

```diff
@@ -1,4 +1,4 @@
-POM_ARTIFACT_ID=apollo-api
+POM_ARTIFACT_ID=apollo-api-jvm
```

Comment:
is it intentionally? I thought we keep the name of artifact the same.

Reply:
I was just trying stuff. I reverted this soon after pushing.